internal/pkg/policy/parsed_policy_test_data.go

```diff
@@ -0,0 +1,856 @@
+
+const noESOutputPolicy = `
```

Comment:
It seems this test is to check for not having an Elasticsearch output. Why do we add the full list of inputs to it? It seems to make the test example unnecessary complex / long?

Reply:
I wanted to keep the additional values on the policy to make sure the parsing code respected them without ES on the policy. This should serve as smoke test when adjusting the logstash transformation logic in the future.

Reply:
I understand if we keep a few lines for sanity check, but do we need all 300 additional ones?

Reply:
I axed two of the inputs, should be more reasonable now.